### DISCUSSION TITLE
secure selfhosted rdns

### DISCUSSION THREAD

**Discussion by suricloud** [serverless-dns/serverless-dns#183] [General]
Hi,

When self hosted on cloudflare and using own domain like rdns.mydomain.com it is open for public.
Can you make it password protected to login and make the blocklist configuration setup.

Also can you implement to have a doh url created for a specific device id ?

And lastly make possible to see the dns logs which are blocked and bypassed.

**Comment by ignoramous** [comment]
> And lastly make possible to see the dns logs which are blocked and bypassed.
> Also can you implement to have a doh url created for a specific device id ?

These are planned, but I am not sure when I will find the time to implement it in the next few months: #17 / #9 

> Can you make it password protected to login and make the blocklist configuration setup.

This is already possible, but it may or may not be easy depending: https://github.com/serverless-dns/serverless-dns#auth (I intend to tweak this auth setup further, it isn't final, but it has remained unchanged since December last year).
